test/emu_mps/test_endtoend.py

```diff
@@ -217,7 +221,82 @@ def test_XY_3atomswith_slm() -> None:
     )  # todo, compare against pulser results
 
 
-def test_end_to_end_afm_ring() -> None:
+@pytest.mark.parametrize(
+    "optimize_order",
+    [
+        False,
+        True,
+    ],
+)
+def test_end_to_end_domain_wall_ring(
+    optimize_order: bool,
+) -> None:
+    torch.manual_seed(seed)
+
+    num_qubits = 6
+    seq = pulser_afm_sequence_ring(
```

Comment:
what's the meaning of this sequence? If I interpret it correctly, you start from a "rrrggg" state, and then have omega=0, so the state only acquires a phase as time goes on, which is the time integral of `H[56,56]` over time, considering `H` as a dense matrix. I understand that it's a nice system you can solve analytically, but what does it add to the existing tests?

Reply:
ok, good.